### PR TITLE
Fixed: Parsing Polish language

### PR DESCRIPTION
**Pull request by tenshiak**
#### Database Migration
NO

#### Description
Almost no-one names their files "polish".

#### Todos
- [ ] Tests

#### Issues Fixed or Closed by this PR
None

### CODE REVIEW

**Review by markus101 — CHANGES_REQUESTED** [review]
Tests in `LanguageParserFixture` (I think) need to be added.

**Comment by tenshiak** [comment]
@markus101 Added all possible test cases.

File: src/NzbDrone.Core/Parser/LanguageParser.cs [hunk 1]
```diff
@@ -178,10 +181,13 @@ private static Language RegexLanguage(string title)
...
             if (match.Groups["hebrew"].Success)
                 return Language.Hebrew;
+            
+            if (match.Groups["polish"].Success)
+                return Language.Polish;
 
             if (match.Groups["chinese"].Success)
                 return Language.Chinese;
-
+
```
Comment (markus101) [inline src/NzbDrone.Core/Parser/LanguageParser.cs:190]: Nit: Remove the spaces on the blank line.

**Review by markus101 — CHANGES_REQUESTED** [review]
Just one more minor thing then good to go.

File: src/NzbDrone.Core/Parser/LanguageParser.cs (hunk 1 above)
Comment (tenshiak) [reply src/NzbDrone.Core/Parser/LanguageParser.cs]: @markus101 All done

**Comment by tenshiak** [comment]
@markus101 When this could be merged? There's a error in the build, but it looks like some dependency fails, not the code itself, all the tests passed before removing whitespaces.

**Pull request merged**